pyproject.toml

```diff
@@ -1,7 +1,7 @@
 # POETRY CONFIG
 [tool.poetry]
 name = "pyneuphonic"
-version = "1.1.6"
+version = "1.1.7"
```

Comment:
You only need to bump the version when we are releasing something new, if we only change the tests we can leave the version as 1.1.6 as we won't publish anything new to pypi.